src/coreimage.cs

```diff
@@ -5452,6 +5252,22 @@ partial interface CIImageRepresentationKeys {
 		[TV (12, 0), iOS (12, 0), Mac (10, 14)]
 		[Field ("kCIImageRepresentationPortraitEffectsMatteImage")]
 		NSString PortraitEffectsMatteImageKey { get; }
+
+		[iOS (13,0)][TV (13,0)][Watch (6,0)][Mac (10,15)]
+		[Field ("kCIImageRepresentationAVSemanticSegmentationMattes")]
+		NSString AVSemanticSegmentationMattesKey { get; }
+
+		[iOS (13,0)][TV (13,0)][Watch (6,0)][Mac (10,15)]
+		[Field ("kCIImageRepresentationSemanticSegmentationSkinMatteImage")]
+		NSString SemanticSegmentationSkinMatteImageKey { get; }
+
+		[iOS (13,0)][TV (13,0)][Watch (6,0)][Mac (10,15)]
+		[Field ("kCIImageRepresentationSemanticSegmentationHairMatteImage")]
+		NSString SemanticSegmentationHairMatteImageKey { get; }
+
+		[iOS (13,0)][TV (13,0)][Watch (6,0)][Mac (10,15)]
+		[Field ("kCIImageRepresentationSemanticSegmentationTeethMatteImage")]
+		NSString SemanticSegmentationTeethMatteImageKey { get; }
```

Comment:
These look like they could be a smart enum.

Reply:
no, but it's part of the type used for a strong dictionary: `CIImageRepresentationOptions`

I'll look if the types are documented, we are already missing a few of those

Reply:
yes, headers (not web doc) do mention the types of those (and the one we were missing from last year)